build.rs

```diff
@@ -1,10 +1,15 @@
 fn main() -> Result<(), Box<dyn std::error::Error>> {
-    // tonic_build::compile_protos("src/proto/blocks.proto")?;
     let mut builder = tonic_build::configure();
 
     // Custom type attributes required for malachite
     builder = builder.type_attribute("snapchain.ShardHash", "#[derive(Eq, PartialOrd, Ord)]");
 
-    builder.compile(&["src/proto/blocks.proto"], &["src/proto"])?;
+    builder.compile(&[
+        "src/proto/blocks.proto",
+        "src/proto/rpc.proto",
+        "src/proto/message.proto",
+        "src/proto/username_proof.proto",
+    ], &["src/proto"])?;
```

Comment:
Would be nice to autodiscover all the files, but not a priority

Reply:
Added a TODO

Reply:
Good to see this implemented as proposed in issue 5 some days ago
https://github.com/farcasterxyz/snapchain-v0/issues/5#